src/Snppts/Snippets/PrettyFaceApp

```diff
@@ -0,0 +1,30 @@
+public class PrettyFaceApp : IAmASnippet
+{
+    public string Slug => "pretty-face-app";
+    public string Title => "Pretty Face App";
+    public string GithubRepoName => "InnovoTecnologias/PrettyFaceApp";
+    public bool ContainsAndroidSample => true;
+    public bool ContainsiOSSample => true;
+    public bool ContainsUWPSample => false;
+
+    public string Description => "This snippet was created to give you a complete starting point for your app.";
+
+    public IAmAnAuthor AuthorInfo => new InnovoTecnologias();
+    
+    public Uri ExternalUri => new Uri("https://github.com/InnovoTecnologias/PrettyFaceApp");
```

Comment:
Please use the `GitHubRepoInfo` property

Reply:
Done.